card-auth-server-acs/src/main/java/com/razorpay/ffs/cas/acs/constant/VISAConstant.java

```diff
@@ -50,13 +45,5 @@ public static VISATransactionStatusInfo getInstance(TransactionStatus transactio
             }
             return result;
         }
-
-        public String getTxnStatusCode() {
```

Comment:
Earlier, we didn't had getter for TransactionStatus, but with @getter will have that.. is that fine?

Reply:
absolutely, No harm in having extra getter. We might don't use it right now. Without getter we can't read status as it's private